lib/components/form/settings-preview.js

```diff
@@ -22,11 +23,16 @@ class SettingsPreview extends Component {
   }
 
   static defaultProps = {
-    editButtonText: <i className='fa fa-pencil' />
+    editButtonText: <i className='fa fa-pencil' />,
+    messages: {
+      // Label accepted as two-element array or single (not array) string literal.
```

Comment:
I think this comment needs to say that each array entry is rendered as a separate line for the button caption.

Reply:
Or, you could use this: https://stackoverflow.com/a/55466235